setup.py

```diff
@@ -43,7 +43,7 @@
 ]
 
 runtime_extras = {
-    'kfp-tekton': ['kfp-tekton~=1.1.1', ],
+    'kfp-tekton': ['kfp-tekton~=1.2.0', ],
```

Comment:
This is effectively a cap of `<1.3` and probably warrants a comment as to why it exists.